src/services/index.js

```diff
@@ -18,10 +18,10 @@ export const callApi = (endpoint, {
     return json;
   });
 
-export const createDemo = (name, email) =>
+export const createDemo = () =>
   callApi('demos', {
     method: 'POST',
-    body: { name, email },
+    body: { name: 'demo' },
```

Comment:
do you use the name for anything?

I guess you kept it cause it is required in the Controller API.

if you don't use it we can remove it from the API too.